superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -316,6 +326,37 @@ const ResultSet = ({
         user?.roles,
       );
```

Comment:
I'm not sure if we missed an upgrade step in our installation but the view was named `SqlLab` instead of `SQLLab` in our database and we had to run `flask fab security-converge`.